release-notes/Cloud/DCP_change_log.md

```diff
@@ -31,7 +31,7 @@ DataMiner Systems are now shown on dataminer.services even if remote access is d
 
 #### 13 February 2024 - Fix - Catalog versions displayed in wrong order [ID_38762]
 
-The versions of a catalog record will now be sorted correctly.
+The versions of a catalog item will now be sorted correctly.
```

Comment:
```suggestion
The versions of a catalog item will now be sorted by using natural sorting instead of default string comparison.
```

Reply:
to technical for users imo, not gonna apply